docs/content/functioneel/zaaktypen_api.md

```diff
@@ -0,0 +1,182 @@
+| Bijzonderheden  | -                                                    |
+
+Deze API omvat (vooralsnog) de resources Catalogussen, Zaaktypen, Informatieobjecttypen, Roltypen, Statustypen en Zaakobjecten.
+
+Op dit moment bieden de resources alleen de mogelijkheid tot raadplegen van de gegevens (GET).
+
+### Resource: catalogus
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil een lijst van alle beschikbare CATALOGUSsen of een specifieke catalogus opvragen.         |
+| *Toevoegen*    |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Verwijderen*  |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Raadplegen*   |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | Een lijst met beschikbare catalogussen wordt teruggegeven aan de consumer bij het opvragen van de lijst.<br>Een specificieke catalogus wordt teruggegeven aan de consumer. |
+| Gegevens       | objecttype CATALOGUS inclusief een lijst met relaties naar daarin gedefinieerde ZAAKTYPEn.<br>Zie de volgende tabel voor de gegevens van deze resource                                 |
+| Specificaties  | https://ref.tst.vng.cloud/ztc/api/v1/schema/#tag/catalogus                                                                                                   |
+| Samenhang      | nvt. |
+| Bijzonderheden | nvt. |
+        
+
+
+### Resource: zaaktype
+
+| **Aspect**     | **Beschrijving**                                                                                                                                                                 |
+|----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Aanleiding     | Een consumer wil (een lijst van beschikbare) zaaktype informatie opvragen uit de catalogus.            |
+| *Toevoegen*    |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Wijzigen*     |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Verwijderen*  |      |
+| - voorwaarde   | nvt.         |
+| - gevolg       | nvt.             |
+| *Raadplegen*   |      |
+| - voorwaarde   | Om zaaktype informatie op te kunnen vragen moet de consumer de UUID van de catalogus waarin het zaaktype is opgenomen weten. Deze UUID kan opgevraagd worden met de catalogus bevragingen services.       |
```

Comment:
Ik zou eerder stellen dat de lijst van zaaktypen uit de list/detail van catalogus gebruikt moet worden. Het zelf ineenpuzzelen van de URLs met de parameters zou ik niet aanmoedigen.

Reply:
@ArjanKloosterboer  je slaat de spijker op zijn kop: ik ben een techneut :-)

Reply:
> 
> 
> Ik zou eerder stellen dat de lijst van zaaktypen uit de list/detail van catalogus gebruikt moet worden. Het zelf ineenpuzzelen van de URLs met de parameters zou ik niet aanmoedigen.

Dit was niet bedoeld om zelf URLs in elkaar te knutselen maar meer als discovery service: eerst vragen aan de ZTC component welke catalogussen er zijn, dan de zaaktypen bij de juiste catalogus opvragen.

Reply:
Tekst is inmiddels aangepast.